llvm/test/DebugInfo/X86/selectionDAG-load-sext.mir

```diff
@@ -0,0 +1,54 @@
+
+# RUN: llc -O2 %s -mtriple=x86_64-unkown-linux -start-before=x86-isel -stop-after=x86-isel -o - | FileCheck %s --check-prefix=MIR
+# RUN: llc -O2 %s -start-before=x86-isel -mtriple=x86_64-unkown-linux --filetype=obj -o %t.o 
+# RUN: llvm-dwarfdump %t.o --name Idx | FileCheck %s --check-prefix=DUMP
+
+# MIR: ![[IDX:[0-9]+]] = !DILocalVariable(name: "Idx"
+# MIR-LABEL: bb.0
+# MIR: %{{[0-9a-zA-Z]+}}{{.*}} = MOVSX64rm32 ${{.*}}, 1, $noreg, @GlobArr, $noreg, debug-instr-number [[INSTR_NUM:[0-9]+]]
+# MIR-NEXT: DBG_INSTR_REF ![[IDX]], !DIExpression(DW_OP_LLVM_arg, 0), dbg-instr-ref([[INSTR_NUM]], 0)
+# MIR-NEXT DBG_INSTR_REF ![[IDX]], !DIExpression(DW_OP_LLVM_arg, 0, DW_OP_LLVM_fragment, 0, 32), dbg-instr-ref([[INSTR_NUM]], 0)
```

Comment:
Missing `:` after MIR-NEXT

But more broadly, why do we emit two DBG_INSTR_REFs?

Reply:
Explained here:

https://github.com/llvm/llvm-project/pull/169779/files#r2567773090

But just to mention again, the code adds a new `SDDbgValue` I don't see a way to modify the old one, and I don't see an easy way to delete an old one, so the old one without the fragment expr remains, and a new one with the fragment expr is also added